app/pages/search/searchPageSelector.spec.js

```diff
@@ -0,0 +1,33 @@
+import { expect } from 'chai';
+
+import { getState } from 'utils/testUtils';
+import searchPageSelector from './searchPageSelector';
+
+describe('pages/search/searchPageSelector', () => {
+  const resources = {
+    'r-1': { id: 'r-1' },
+    'r-2': { id: 'r-2' },
+    'r-3': { id: 'r-3' },
+  };
+  const searchFilters = { search: 'search text' };
+  const searchResults = ['r-1', 'r-3'];
+  const state = getState({
+    'data.resources': resources,
+    'searchPage.searchFilters': searchFilters,
+    'searchPage.searchResults': searchResults,
+  });
+  const selected = searchPageSelector(state);
```

Comment:
- [ ] `getState` and the selector should be executed inside the tests. Otherwise if they raise errors no tests will be run and the errors won't show which tests failed. So you could for example do `function getSelected() { const state = getState(...); return searchPageSelector(state); }` and then have the tests be like `expect(getSelected().isFetching).to.exist`.